sv_pipeline/genome/load_data_tests.py

```diff
@@ -0,0 +1,295 @@
+                     genotypes=[hl.struct(sample_id='SAMPLE-1', gq=999, num_alt=0, cn=hl.null('int')),
+                                hl.struct(sample_id='SAMPLE-3', gq=999, num_alt=0, cn=hl.null('int')),
+                                hl.struct(sample_id='SAMPLE-5', gq=1, num_alt=1, cn=hl.null('int')),])
+
+TEST_PASSWORD = 'ExamplePasswd'
+TEST_MT_PATH = 'test_mt/mts'
+TEST_GENCODE_PATH = 'test_gtf/gtf'
+TEST_HOST = 'TEST_HOST'
+TEST_PORT = '9500'
+TEST_GENCODE_RELEASE = 31
+TEST_BLOCK_SIZE = 3000
+TEST_NUM_SHARDS = 10
+
+
+class LoadDataTest(unittest.TestCase):
+    def setUp(self):
+        self.vcf_file = tempfile.mkstemp(suffix='.vcf')[1]
+        with open(self.vcf_file, 'w') as f:
+            f.writelines('\n'.join(VCF_DATA))
+        hl.init()
+        self.mt = hl.import_vcf(self.vcf_file, reference_genome='GRCh38')
+
+    def tearDown(self):
+        hl.stop()
+        os.remove(self.vcf_file)
+
+    @mock.patch('sv_pipeline.genome.load_data.get_sample_subset')
+    @mock.patch('sv_pipeline.genome.load_data.logger')
+    @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.gene_id_mapping', hl.literal(GENE_ID_MAPPING))
+    def test_sub_and_annotation(self, mock_logger, mock_get_sample):
+        # Test subsetting dataset
+        mock_get_sample.return_value = {'SAMPLE-1', 'SAMPLE-3', 'SAMPLE-5', 'SAMPLE-6'}
+        with self.assertRaises(Exception) as e:
+            _ = sub_setting_mt('test_guid', self.mt, sample_type='WGS', skip_sample_subset=False,
+                                  ignore_missing_samples=False)
+        self.assertEqual(str(e.exception), 'Missing the following 1 samples:\nSAMPLE-6')
+
+        mock_get_sample.return_value = {'SAMPLE-1', 'SAMPLE-3', 'SAMPLE-5'}
+        rows = sub_setting_mt('test_guid', self.mt, sample_type='WGS', skip_sample_subset=False, ignore_missing_samples=True)
+        calls = [
+            mock.call('Missing the following 0 samples:\n'),
+            mock.call('Subsetting to 3 samples'),
+        ]
+        mock_logger.info.assert_has_calls(calls)
+        mock_get_sample.assert_called_with('test_guid', 'WGS')
+        self.assertEqual(rows.count(), 5)
+
+        # Test annotate required fields
+        rows = annotate_fields(rows)
+        row_list = rows.take(5)
+
+        self.assertListEqual([row_list[0], row_list[2], row_list[4]], hl.eval([VARIANT0, VARIANT2, VARIANT4]))
+
+    @mock.patch('sv_pipeline.genome.load_data.logger')
+    @mock.patch('sv_pipeline.genome.load_data.hl')
+    @mock.patch('sv_pipeline.genome.load_data.load_gencode')
+    @mock.patch('sv_pipeline.genome.load_data.os')
+    @mock.patch('sv_pipeline.genome.load_data.time.time')
+    @mock.patch('sv_pipeline.genome.load_data.sub_setting_mt')
+    @mock.patch('sv_pipeline.genome.load_data.annotate_fields')
+    @mock.patch('sv_pipeline.genome.load_data.ElasticsearchClient')
+    def test_main(self, mock_es, mock_annot, mock_subset, mock_time, mock_os, mock_gencode, mock_hl, mock_logger):
```

Comment:
As I explained in my previous PR review, this function should mock out all the helpers in the file - `load_mt`, `subset_mt`, `annotate_fields ` and `export_to_es`. It should be a very short test, that essentially just checks that those mocks are called as expected to make sure argument parsing is working. Then you should write test for each of those helpers independently. This giant test with tons of mocks is impossible to read as it is